packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -922,132 +924,115 @@ const runGenerator = async (
         sync: true
     })
 
-    if (extend) {
-        // Bootstrap the projects.
-        getFiles(BOOTSTRAP_DIR)
-            .map((file) => file.replace(BOOTSTRAP_DIR, ''))
-            .forEach((relFilePath) =>
-                processTemplate(relFilePath, BOOTSTRAP_DIR, outputDir, context)
-            )
-
-        // Copy required assets defined on the preset level.
-        const {assets = []} = preset
-        assets.forEach((asset) => {
-            sh.cp('-rf', p.join(packagePath, asset), outputDir)
-        })
-    } else {
-        // Copy the base template either from the package or npm.
-        sh.cp('-rf', p.join(packagePath, '*'), outputDir)
-
-        // Copy template specific assets over.
-        const assetsDir = p.join(ASSETS_TEMPLATES_DIR, id)
-        if (sh.test('-e', assetsDir)) {
-            getFiles(assetsDir)
-                .map((file) => {
-                    const relFilePath = file.replace(assetsDir, '')
-                    return relFilePath
-                })
-                .forEach((relFilePath) => {
-                    processTemplate(relFilePath, assetsDir, outputDir, context)
-                })
-        }
-
-        // Check project type and handle appropriately
-        if (answers.project.type === 'PWAKitAppExtensionProject') {
-            const devOutputDir = p.join(outputDir, LOCAL_DEV_PROJECT_DIR)
+    // Copy the base template either from the package or npm.
+    sh.cp('-rf', p.join(packagePath, '{*,.*}'), outputDir)
```

Comment:
Windows wasn't copying the hidden files (e.g. .eslintrc) into the output folder so lint was failing amongst other things